inception-project/inception
# Feature/2260 create log file in inception folder by default

**What's in the PR**
- Add logger which captures recent log messages in memory
- Add option to open a log window from the tray icon
- Create rolling log file in the inception home folder when running with the embedded application server

**How to test manually**
* Check log accessible via try
* Check log written to application home

**Automatic testing**

**Documentation**
